etc/visual-studio/libopenthread-ncp-uart.vcxproj

```diff
@@ -73,4 +76,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
This PR actually removes the newline.  Are you using Visual Studio to edit these files?